doc/neps/nep-0036-fair-play.rst

```diff
@@ -0,0 +1,150 @@
+==================
+NEP 36 — Fair Play
+==================
+
+:Author: Stéfan van der Walt <stefanv@berkeley.edu>
+:Status: Draft
+:Type: Standards Track
+:Created: 2019-10-24
+:Resolution: <url> (required for Accepted | Rejected | Withdrawn)
+
+
+Abstract
+--------
+
+This document describes Rules of Play for engaging with the NumPy
+project as an external entity.  It discusses usage of the name NumPy,
+modified distributions of NumPy, and how to best communicate
+with the project when patched versions are deployed.
+
+Companies and developers will know after reading this NEP what kinds
+of behavior the NumPy community consider troublesome, bothersome, and
```

Comment:
propose "... community would like to see, and which it considers troublesome, ..."